README.md

```diff
@@ -1,4 +1,7 @@
-[![Code Climate](https://codeclimate.com/github/panorama-ed/order_as_specified/badges/gpa.svg)](https://codeclimate.com/github/panorama-ed/order_as_specified) [![Test Coverage](https://codeclimate.com/github/panorama-ed/order_as_specified/badges/coverage.svg)](https://codeclimate.com/github/panorama-ed/order_as_specified) [![Build Status](https://travis-ci.org/panorama-ed/order_as_specified.svg)](https://travis-ci.org/panorama-ed/order_as_specified) [![Inline docs](http://inch-ci.org/github/panorama-ed/order_as_specified.png)](http://inch-ci.org/github/panorama-ed/order_as_specified) [![Gem Version](https://badge.fury.io/rb/order_as_specified.svg)](http://badge.fury.io/rb/order_as_specified)
```

Comment:
So I tried to get code coverage to work on Code Climate and couldn't. It seems I might no longer be an account owner for that repo in Code Climate? In any event, we've stopped using Code Climate for our private repos and I don't think it adds much here. I've used Codecov on a [personal project](https://github.com/JacobEvelyn/friends) and knew it was super easy to set up and works really well (look at that diff report in the PR!) so figured I'd give it a shot. Let me know if you don't feel comfortable with this change!